### PR TITLE
[Excel] (troubleshooting) guidance for reporting errors

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Fixes [1786](https://github.com/OfficeDev/office-js-docs-pr/issues/1786)

### CODE REVIEW

File: docs/excel/excel-add-ins-error-handling.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ When an Excel JavaScript API request fails to run successfully, the API returns
 - **debugInfo**: When present, the `debugInfo` property of the error message provides additional information that you can use to understand the root cause of the error.
 
 > [!NOTE]
-> If you use `console.log()` to print error messages to the console, those messages will only be visible on the server. End users will not see those error messages in the add-in task pane or anywhere in the Office application.
+> If you use `console.log()` to print error messages to the console, those messages will only be visible on the server. End users will not see those error messages in the add-in task pane or anywhere in the Office application. To report errors to the user, see [Error notifications](#error-notifications).
```
Comment (alison-mk) [inline docs/excel/excel-add-ins-error-handling.md:56]: Switching to present simple tense 
```suggestion
> If you use `console.log()` to print error messages to the console, those messages are only visible on the server. End users do not see those error messages in the add-in task pane or anywhere in the Office application. To report errors to the user, see [Error notifications](#error-notifications).
```

File: docs/excel/excel-add-ins-error-handling.md [hunk 2]
```diff
@@ -103,6 +103,12 @@ The following table is a list of errors that the API may return.
 > [!NOTE]
 > The preceding table lists error messages you may encounter while using the Excel JavaScript API. If you are working with the Common API instead of the application-specific Excel JavaScript API, see [Office Common API error codes](../reference/javascript-api-for-office-error-codes.md) to learn about relevant error messages.
 
+## Error notifications
+
+How you report errors to users depends on the UI system you are using. If you're using React as the UI system, use the Fluent UI components and design elements. Pick out an appropriate control on this [Fluent UI page](https://developer.microsoft.com/en-us/fluentui#/controls/web). We recommend that error messages be conveyed with a message bar, dialog, or modal. However, if the error is in the user's input, display the error in bold red near the input control. The sample [Office-Add-in-Microsoft-Graph-React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/0706cc67645675a48747f1fec1b1e5722b575b11/Samples/auth/Office-Add-in-Microsoft-Graph-React) uses a MessageBar, but modifies it to take account of the personality menu in an add-in task pane.
```
Comment (AlexJerabek) [inline docs/excel/excel-add-ins-error-handling.md:108]:
```suggestion
How you report errors to users depends on the UI system you are using. If you're using React as the UI system, use the Fluent UI components and design elements. Pick an appropriate control from this [Fluent UI page](https://developer.microsoft.com/fluentui#/controls/web). We recommend that error messages be conveyed with a message bar, dialog, or modal. If the error is in the user's input, display the error in bold red near the input control. The sample [Office-Add-in-Microsoft-Graph-React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/0706cc67645675a48747f1fec1b1e5722b575b11/Samples/auth/Office-Add-in-Microsoft-Graph-React) uses a MessageBar, but modifies it to take account of the personality menu in an add-in task pane.
```

File: docs/excel/excel-add-ins-error-handling.md [hunk 3]
```diff
@@ -103,6 +103,12 @@ The following table is a list of errors that the API may return.
...
+
+If you're not using React for the UI, consider using the older Fabric UI components implemented directly in HTML/Javascript. Some example templates are in the [Office-Add-in-UX-Design-Patterns-Code](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code/tree/master/templates) repo. Take a look especially in the dialog and navigation subfolders. The sample [Excel-Add-in-SalesLeads](https://github.com/OfficeDev/Excel-Add-in-SalesLeads) uses a message banner.
```
Comment (AlexJerabek) [inline docs/excel/excel-add-ins-error-handling.md:110]:
```suggestion
If you're not using React for the UI, consider using the older Fabric UI components implemented directly in HTML and JavaScript. Some example templates are in the [Office-Add-in-UX-Design-Patterns-Code](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code/tree/master/templates) repo. Take a look especially in the dialog and navigation subfolders. The sample [Excel-Add-in-SalesLeads](https://github.com/OfficeDev/Excel-Add-in-SalesLeads) uses a message banner.
```

**Review by AlexJerabek — APPROVED** [review]
A couple small suggestions. The only one I'd say you have to take is removing the localization string from the URL.

File: docs/excel/excel-add-ins-error-handling.md (hunk 3 above)
Comment (alison-mk) [inline docs/excel/excel-add-ins-error-handling.md]:
```suggestion
If you're not using React for the UI, consider using the older Fabric UI components implemented directly in HTML and Javascript. Some example templates are in the [Office-Add-in-UX-Design-Patterns-Code](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code/tree/master/templates) repository. Take a look especially in the dialog and navigation subfolders. The sample [Excel-Add-in-SalesLeads](https://github.com/OfficeDev/Excel-Add-in-SalesLeads) uses a message banner.
```

File: docs/excel/excel-add-ins-error-handling.md (hunk 2 above)
Comment (alison-mk) [inline docs/excel/excel-add-ins-error-handling.md]:
```suggestion
How you report errors to users depends on the UI system you are using. If you're using React as the UI system, use the Fluent UI components and design elements. Pick out an appropriate control on this [Fluent UI page](https://developer.microsoft.com/en-us/fluentui#/controls/web). We recommend that error messages be conveyed with a message bar, dialog, or modal. However, if the error is in the user's input, display the error in bold red near the input control. The sample [Office-Add-in-Microsoft-Graph-React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/0706cc67645675a48747f1fec1b1e5722b575b11/Samples/auth/Office-Add-in-Microsoft-Graph-React) uses a MessageBar element and modifies it to take account of the personality menu in an add-in task pane.
```

File: docs/excel/excel-add-ins-error-handling.md [hunk 4]
```diff
@@ -103,6 +103,12 @@ The following table is a list of errors that the API may return.
 > [!NOTE]
 > The preceding table lists error messages you may encounter while using the Excel JavaScript API. If you are working with the Common API instead of the application-specific Excel JavaScript API, see [Office Common API error codes](../reference/javascript-api-for-office-error-codes.md) to learn about relevant error messages.
 
+## Error notifications
+
+How you report errors to users depends on the UI system you are using. If you're using React as the UI system, use the Fluent UI components and design elements. Pick an appropriate control from this [Fluent UI page](https://developer.microsoft.com/fluentui#/controls/web). We recommend that error messages be conveyed with a message bar, dialog, or modal. If the error is in the user's input, display the error in bold red near the input control. The sample [Office-Add-in-Microsoft-Graph-React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/0706cc67645675a48747f1fec1b1e5722b575b11/Samples/auth/Office-Add-in-Microsoft-Graph-React) uses a MessageBar, but modifies it to take account of the personality menu in an add-in task pane.
```
Comment (alison-mk) [inline docs/excel/excel-add-ins-error-handling.md:108]:
```suggestion
How you report errors to users depends on the UI system you are using. If you're using React as the UI system, use the Fluent UI components and design elements. Pick an appropriate control from this [Fluent UI page](https://developer.microsoft.com/fluentui#/controls/web). We recommend that error messages be conveyed with a message bar, dialog, or modal. If the error is in the user's input, display the error in bold red near the input control. The sample [Office-Add-in-Microsoft-Graph-React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/0706cc67645675a48747f1fec1b1e5722b575b11/Samples/auth/Office-Add-in-Microsoft-Graph-React) uses a MessageBar element and modifies it to take account of the personality menu in an add-in task pane.
```

File: docs/excel/excel-add-ins-error-handling.md [hunk 5]
```diff
@@ -103,6 +103,12 @@ The following table is a list of errors that the API may return.
...
+
+If you're not using React for the UI, consider using the older Fabric UI components implemented directly in HTML and JavaScript. Some example templates are in the [Office-Add-in-UX-Design-Patterns-Code](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code/tree/master/templates) repo. Take a look especially in the dialog and navigation subfolders. The sample [Excel-Add-in-SalesLeads](https://github.com/OfficeDev/Excel-Add-in-SalesLeads) uses a message banner.
```
Comment (alison-mk) [inline docs/excel/excel-add-ins-error-handling.md:110]:
```suggestion
If you're not using React for the UI, consider using the older Fabric UI components implemented directly in HTML and JavaScript. Some example templates are in the [Office-Add-in-UX-Design-Patterns-Code](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code/tree/master/templates) repository. Take a look especially in the dialog and navigation subfolders. The sample [Excel-Add-in-SalesLeads](https://github.com/OfficeDev/Excel-Add-in-SalesLeads) uses a message banner.
```

**Review by alison-mk — APPROVED** [review]

**Pull request merged**